lang/python/python-babel/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2019-2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-babel
+PKG_VERSION:=2.8.0
+PKG_RELEASE:=1
+
+PYPI_NAME:=Babel
+PKG_HASH:=1aac2ae2d0d8ea368fa90906567f5c08463d98ade155c0c4bfedd6a0f7160e38
+
+PKG_MAINTAINER:=Josef Schlehofer <josef.schlehofer@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-babel
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Internationalization utilities
+  URL:=https://babel.pocoo.org/en/latest/
+  DEPENDS:= \
+    +python3-cgi \
+    +python3-decimal \
+    +python3-distutils \
+    +python3-email \
+    +python3-light \
+    +python3-pytz \
+    +python3-urllib \
+    +python3-xml
```

Comment:
Looks like the xml module is only used in [scripts/import_cldr.py](https://github.com/python-babel/babel/blob/v2.8.0/scripts/import_cldr.py#L21-L24) which appears to be used during compile-time and not run-time, so I don't think `+python3-xml` is necessary here.

(urllib is only explicitly imported in [scripts/download_import_cldr.py](https://github.com/python-babel/babel/blob/v2.8.0/scripts/download_import_cldr.py#L10-L13), but because email depends on urllib (#12942) I think it is okay to keep this dependency.)

Reply:
Thanks @jefferyto. You are right. `python3-xml` is not necessary.